my_list._class.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'my_enumerable_mod'
+
+class Mylist
```

Comment:
- [ ] Please use `CamelCase` on class naming, this class should be named `MyList`, with `L` and not `l` 😄